frontend/packages/noobaa-storage-plugin/src/components/details-card/details-card.tsx

```diff
@@ -15,14 +16,19 @@ import {
 } from '@console/internal/components/dashboard/with-dashboard-resources';
 import { FirehoseResource, ExternalLink, FirehoseResult } from '@console/internal/components/utils';
 import { InfrastructureModel } from '@console/internal/models/index';
-import { SubscriptionModel } from '@console/operator-lifecycle-manager/src/models';
+import {
+  SubscriptionModel,
+  ClusterServiceVersionModel,
+} from '@console/operator-lifecycle-manager/src/models';
 import { referenceForModel, K8sResourceKind } from '@console/internal/module/k8s';
 import { PrometheusResponse } from '@console/internal/components/graphs';
 import { useK8sGet } from '@console/internal/components/utils/k8s-get-hook';
 import { getOCSVersion } from '@console/ceph-storage-plugin/src/selectors';
 import { RGW_FLAG } from '@console/ceph-storage-plugin/src/features';
 import { getMetric } from '../../utils';
 import './details-card.scss';
+import { resourcePathFromModel } from '@console/internal/components/utils/resource-link';
+import { CEPH_STORAGE_NAMESPACE } from '@console/ceph-storage-plugin/src/constants/index';
```

Comment:
```suggestion
import { RGW_FLAG } from '@console/ceph-storage-plugin/src/features';
import { resourcePathFromModel } from '@console/internal/components/utils/resource-link';
import { CEPH_STORAGE_NAMESPACE } from '@console/ceph-storage-plugin/src/constants/index';
import { getMetric } from '../../utils';
import './details-card.scss';
```
We maintain the order as following:
1) package/vendor imports
2) @console imports 
3) relative imports (Ex: '../..')